src/ApiService/ApiService/QueueWebhooks.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Microsoft.Azure.Functions.Worker;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+
+namespace Microsoft.OneFuzz.Service;
+
+public class QueueWebhooks
+{
+    private readonly ILogTracerFactory _loggerFactory;
+
+    public QueueWebhooks(ILogTracerFactory loggerFactory)

```

Comment:
we do it a bit differently now

see:

https://github.com/microsoft/onefuzz/blob/e32f42cc1bb4a5478449a406d85a9e6125642c15/src/ApiService/ApiService/QueueFileChanges.cs#L20